pkg/iris/api.go

```diff
@@ -44,28 +44,29 @@ func (a *api) GetConfiguration(configurationName string, accessToken string) ([]
 	return responseData, nil
 }
 
-func (a *api) PostResults(results []byte, accessToken string) (string, error) {
+func (a *api) PostResults(results []byte, accessToken string) error {
 	resultUrl := fmt.Sprintf("https://%s/services/vsm-iris/v1/results", a.uri)
 	postReq, err := http.NewRequest("POST", resultUrl, nil)
 	postReq.Header.Set("Content-Type", "application/json")
 	postReq.Header.Set("Authorization", "Bearer "+accessToken)
 	postReq.Body = ioutil.NopCloser(bytes.NewBuffer(results))
 	if err != nil {
-		log.Infof("Post results request: Error while posting results: %s", resultUrl)
-		return postReq.URL.String(), err
+		log.Errorf("Error while posting results: %v", err)
```

Comment:
We should make sure, that we initiated the logger beforehand with some meaningful context/meta information so that we also get maybe the run id or the workspace id for this log line for json logging

Reply:
ok, can we create a follow up ticket for this or you recommend to do it in same ticket.

Note: I may need to check how its done in Go.

Reply:
found something called `github.com/sirupsen/logrus`. I can check if it can be done fast.